static/app/utils.tsx

```diff
@@ -339,3 +339,15 @@ export function escapeDoubleQuotes(str: string) {
 export function generateBaseControlSiloUrl() {
   return ConfigStore.get('links').sentryUrl || '';
 }
+
+export function generateIntegrationInstallLink(orgSlug) {
+  const sentryDomain = window.__initialData.links.sentryUrl.split('/')[2];
+  return (
+    window.location.protocol +
+    '//' +
+    orgSlug +
+    '.' +
+    sentryDomain +
+    window.location.pathname
+  );
```

Comment:
Let's prefer template strings
```suggestion
  return `${window.location.protocol}//${orgSlug}.${sentryDomain}${window.location.pathname}`;
```